src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -47,7 +47,7 @@ enum {
                                                    necessarily disconnected yet */
 };
 
-UCS_CLASS_DECLARE(uct_rdmacm_ep_t, const uct_ep_params_t *);
+UCS_CLASS_DECLARE(uct_rdmacm_cm_ep_t, const uct_ep_params_t *);
```

Comment:
why do we even need this CLASS_DECLARE?
it's not that any other class inherits from it
and UCX worked even when this line was wrong, no?

Reply:
you mean why we are using classes to create/manage the ep?

Reply:
i mean UCS_CLASS_DEFINE in C file should be enough in this case